spring-webflux/src/test/java/org/springframework/web/reactive/socket/AbstractReactiveWebSocketIntegrationTests.java

```diff
@@ -88,10 +89,13 @@ abstract class AbstractReactiveWebSocketIntegrationTests {
 	@interface ParameterizedWebSocketTest {
 	}
 
+	private static final JettyWebSocketClient jettyClient = new JettyWebSocketClient();
+
 	static Stream<Object[]> arguments() throws IOException {
 
 		WebSocketClient[] clients = new WebSocketClient[] {
 				new TomcatWebSocketClient(),
+				jettyClient,
```

Comment:
The Jetty WebSocketClient is the only one which needs to be started and stopped. Because of this if we create a new one every time the tests can eventually run out of memory because they are never stopped.

Reply:
would feel nicer to come up with a way to stop after every test, but if that is too difficult then do this, but add a comment why.

Reply:
I found a better way to do this. There is `org.springframework.context.Lifecycle` which I can make `JettyWebSocketClient` implement.